ui_tests/pages/inventory_page.py

```diff
@@ -0,0 +1,336 @@
+import time
+
+import pages.helper as helper
+import selenium.common.exceptions
+from logger.logger import Logger
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from webdriver.webriver_factory import WebDriverFactory
+
+logger = Logger().get_logger()
+driver = WebDriverFactory.get_driver()
+
+
+class InventoryPage:
+    def checkIfInventoryTableIsDisplayed(self):
+        logger.info("Check if groups page is displayed")
+        inventory_table_xpath = "//div[@data-test='sc4snmp:inventory-table']"
+        inventory_container = driver.find_element(By.XPATH, inventory_table_xpath)
+        return inventory_container.is_displayed()
+
+    def check_if_entry_is_on_list(self, host_ip):
+        logger.info(f"Checking if host/group entry is configured (is on list)")
+        entries_on_list_xpath = "//td[@data-test='sc4snmp:inventory-address']"
+        entries = driver.find_elements(By.XPATH, entries_on_list_xpath)
+        logger.info(f"list length: {len(entries)}")
+        for el in entries:
+            logger.info(f"entry name >  |{el.text}|")  # debug
```

Comment:
Similar comment about commenting this line the same way as it is done [here](https://github.com/splunk/splunk-connect-for-snmp/pull/914/files#diff-2b38a6d07be742a0e7a28656e97622a3f8354667e8645243c08afe21245b76c3R123)

Reply:
done